metagraph/src/annotation/representation/annotation_matrix/static_annotators_def.hpp

```diff
@@ -61,6 +64,9 @@ template <>
 inline const std::string RowDiffBRWTAnnotator::kExtension = ".row_diff_brwt.annodbg";
 template <>
 inline const std::string RowDiffRowSparseAnnotator::kExtension = ".row_diff_sparse.annodbg";
+template <>
+inline const std::string ExtendedMultiBRWTAnnotator::kExtension = ".count_brwt.annodbg";
```

Comment:
Wouldn't it be much more extensible to have counts be stored separately as an int_vector?

Reply:
Also answering to this:
> Instead of making this specific to BRWT, why don't we instead introduce a trivial implementation of `get_row_ranks` for `BinaryMatrix` and allow this work for all annotation types?

Only column-major annotations support rank on columns, so we can't implement `get_row_ranks` for all annotations.
For instance, the `row_flat` annotation would call rank on rows, hence, the layout of the k-mer counts would have to be different as well.

I thought about storing the counts always separately, but saw no advantages of that right now.
On the other hand, keeping them together with the matrix is easier because it never becomes out of sync.
If there is a need, we can easily change this later.